language/api/Analyze/Program.cs

```diff
@@ -138,17 +147,23 @@ private static void AnalyzeSyntaxFromText(string text)
                 Type = Document.Types.Type.PlainText
             },
             new Features() { ExtractSyntax = true });
-            WriteSentences(response.Sentences);
+            WriteSentences(response.Sentences, response.Tokens);
         }
 
         // [START analyze_syntax_from_file]
-        private static void WriteSentences(IEnumerable<Sentence> sentences)
+        private static void WriteSentences(IEnumerable<Sentence> sentences,
+            RepeatedField<Token> tokens)
         {
             Console.WriteLine("Sentences:");
             foreach (var sentence in sentences)
             {
                 Console.WriteLine($"\t{sentence.Text.BeginOffset}: {sentence.Text.Content}");
             }
+            Console.WriteLine("Tokens:");
+            foreach (var token in tokens)
+            {
+                Console.WriteLine($"{token.PartOfSpeech.Tag} {token.Text.Content}");

```

Comment:
Github makes it impossible to tell, but make sure all lines are < 80 characters.  Then, merge away!